cfgov/v1/atomic_elements/organisms.py

```diff
@@ -664,22 +664,48 @@ class Meta:
 
 
 class ChartBlock(blocks.StructBlock):
-    element_id = blocks.CharBlock(
+    title = blocks.CharBlock(required=True)
+    # todo: make radio buttons
+    chart_type = blocks.ChoiceBlock(choices=[
+        ('bar', 'Bar'),
+        ('line', 'Line'),
+        ('tile_map', 'Tile Map'),
```

Comment:
Whoopsie. Would it be easier to change it in `cfpb-chart-builder`? https://github.com/cfpb/cfpb-chart-builder/blob/master/src/index.js#L41